test/perf/run.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
```

Comment:
Was validating the script on linux. Came accross this. Not sure if this is expected. Attached log file.
(Not really important, unsure why it says unsupported os, the if logic looks sound)

[20230823-15h08m29s.log](https://github.com/mojaloop/central-services-stream/files/12422992/20230823-15h08m29s.log)

```
2023-08-23T20:12:45.038Z - [31merror[39m: Consumer::consume()::syncQueue.queue[0] - workDoneCb - error: Error: KafkaConsumer is disconnected
2023-08-23T20:12:45.038Z - [31merror[39m: Consumer::onError()[topics='test'] - Error: KafkaConsumer is disconnected
    at KafkaConsumer.commitMessageSync (/home/kleyow/mojaloop/central-services-stream/node_modules/node-rdkafka/lib/kafka-consumer.js:581:32)
    at Consumer.commitMessageSync (/home/kleyow/mojaloop/central-services-stream/src/kafka/consumer.js:786:20)
    at /home/kleyow/mojaloop/central-services-stream/test/perf/scripts/consumer.js:135:29
    at new Promise (<anonymous>)
    at /home/kleyow/mojaloop/central-services-stream/test/perf/scripts/consumer.js:111:16
    at /home/kleyow/mojaloop/central-services-stream/src/kafka/consumer.js:473:25
    at /home/kleyow/mojaloop/central-services-stream/node_modules/async/dist/async.js:4016:13
    at Object.process (/home/kleyow/mojaloop/central-services-stream/node_modules/async/dist/async.js:1679:21)
    at /home/kleyow/mojaloop/central-services-stream/node_modules/async/dist/async.js:1531:23
    at /home/kleyow/mojaloop/central-services-stream/node_modules/async/dist/async.js:74:45)
2023-08-23T20:12:45.038Z - [31merror[39m: Consumer::_consumerRecursive()::syncQueue.Single.push - error: Error: KafkaConsumer is disconnected
2023-08-23T20:12:45.038Z - [31merror[39m: Consumer::consume() - onRecursive - error Error: KafkaConsumer is disconnected
2023-08-23T20:12:45.039Z - [31merror[39m: uncaughtException: Local: Erroneous state
Error: Local: Erroneous state
    at Function.createLibrdkafkaError [as create] (/home/kleyow/mojaloop/central-services-stream/node_modules/node-rdkafka/lib/error.js:456:10)
    at KafkaConsumer.Client._errorWrap (/home/kleyow/mojaloop/central-services-stream/node_modules/node-rdkafka/lib/client.js:481:29)
    at KafkaConsumer.resume (/home/kleyow/mojaloop/central-services-stream/node_modules/node-rdkafka/lib/kafka-consumer.js:641:15)
    at /home/kleyow/mojaloop/central-services-stream/src/kafka/consumer.js:484:24
    at /home/kleyow/mojaloop/central-services-stream/node_modules/async/dist/async.js:1496:46
    at Array.forEach (<anonymous>)
    at trigger (/home/kleyow/mojaloop/central-services-stream/node_modules/async/dist/async.js:1496:27)
    at /home/kleyow/mojaloop/central-services-stream/node_modules/async/dist/async.js:1569:21
    at /home/kleyow/mojaloop/central-services-stream/node_modules/async/dist/async.js:327:20
    at /home/kleyow/mojaloop/central-services-stream/src/kafka/consumer.js:478:11
2023-08-23T20:12:45.039Z - [31merror[39m: uncaughtException: Local: Erroneous state
Error: Local: Erroneous state
    at Function.createLibrdkafkaError [as create] (/home/kleyow/mojaloop/central-services-stream/node_modules/node-rdkafka/lib/error.js:456:10)
    at KafkaConsumer.Client._errorWrap (/home/kleyow/mojaloop/central-services-stream/node_modules/node-rdkafka/lib/client.js:481:29)
    at KafkaConsumer.resume (/home/kleyow/mojaloop/central-services-stream/node_modules/node-rdkafka/lib/kafka-consumer.js:641:15)
    at /home/kleyow/mojaloop/central-services-stream/src/kafka/consumer.js:484:24
    at /home/kleyow/mojaloop/central-services-stream/node_modules/async/dist/async.js:1496:46
    at Array.forEach (<anonymous>)
    at trigger (/home/kleyow/mojaloop/central-services-stream/node_modules/async/dist/async.js:1496:27)
    at /home/kleyow/mojaloop/central-services-stream/node_modules/async/dist/async.js:1569:21
    at /home/kleyow/mojaloop/central-services-stream/node_modules/async/dist/async.js:327:20
    at /home/kleyow/mojaloop/central-services-stream/src/kafka/consumer.js:478:11
```

Reply:
I think we need to jump on a call to discuss this, but I believe that a "state" issue with regard to the offset.

I.e. the offset that your client thinks its at is not the same that its trying to commit.

Reply:
Also, I have not been able to reproduce it on my machine.

Reply:
Non-issue. Linux apt installs older version of rdkafka.